src/nginxconfig/i18n/en/templates/domain_sections/presets.js

```diff
@@ -30,4 +30,5 @@ export default {
     frontend: 'Frontend',
     nodeJs: 'Node.js',
     singlePageApplication: 'Single-page application',
+    rails: 'Rails',
```

Comment:
Ditto -- Can you bump the copyright year in this file as we're touching it, please?